workers/ohsome_quality_analyst/api.py

```diff
@@ -173,9 +190,18 @@ async def _fetch_report(
 
 
 @app.get("/geometries/{dataset}")
-async def get_bpolys_from_db(dataset: str, featureId: int):
-    bpolys = await db_client.get_bpolys_from_db(dataset=dataset, feature_id=featureId)
-    return bpolys
+async def get_bpolys_from_db(
```

Comment:
Tests are not sufficient for this function. But I will remove the function in a separate PR since it is not in use.